### PR TITLE
fix for: Raven can get caught in an alert loop

### PR DESCRIPTION
**Pull request by efjacobson**

### CODE REVIEW

**Comment by efjacobson** [comment]
@gdbrown lmk what you think now

File: src/plugins/raven/sagas/connectionFlow.js [hunk 1]
```diff
@@ -88,6 +77,16 @@ export default function* (raven) {
       yield take(connectsChannel);
       attempts = 0;
     } catch (e) {
+      yield swal.fire({
```
Comment (gdbrown) [inline src/plugins/raven/sagas/connectionFlow.js:80]: @efjacobson i would not alert until after 10 attempts

**Review by gdbrown — CHANGES_REQUESTED** [review]

File: src/plugins/raven/sagas/connectionFlow.js (hunk 1 above)
Comment (efjacobson) [reply src/plugins/raven/sagas/connectionFlow.js]: done

**Comment by efjacobson** [comment]
@gdbrown seems to me that the way it is now, after the 10th failed attempt, the swal will fire and after that nothing will happen as it will just continue forevermore

**Review by gdbrown — APPROVED** [review]

**Pull request merged**